### PR TITLE
Refactor adding entries to transport

### PR DESCRIPTION
**Pull request by mbtools**
- Move `RS_CORR_INSERT` to CTS API class
- Replace all calls to `RS_CORR_INSERT` with CTS API method

### CODE REVIEW

File: src/objects/zcl_abapgit_object_para.clas.abap [hunk 1]
```diff
@@ -79,35 +79,22 @@ CLASS zcl_abapgit_object_para IMPLEMENTATION.
...
-      zcx_abapgit_exception=>raise_t100( ).
-    ENDIF.
+    DELETE FROM tpara WHERE paramid = lv_paramid.
+    DELETE FROM tparat WHERE paramid = lv_paramid.
+
+    CALL FUNCTION 'RS_TREE_OBJECT_PLACEMENT'
+      EXPORTING
+        object    = lv_paramid
+        operation = 'DELETE'
+        type      = 'CR'.
```
Comment (larshp) [inline src/objects/zcl_abapgit_object_para.clas.abap:97]: catch exception and call unlock() ?

Comment (larshp) [reply src/objects/zcl_abapgit_object_para.clas.abap:97]: alternatively move the unlock to before the CTS calls?

**Comment by mbtools** [comment]
Waiting for CI run

**Review by larshp — APPROVED** [review]

**Comment by mbtools** [comment]
CI caught issue with XINX deletion. Reverted back to the original call to `RS_CORR_INSERT`.

**Pull request merged**